README.md

```diff
@@ -4,7 +4,7 @@
 > I created this project by first designing it using wireframes.
 
 # Live View
-> To view this project live, please visit this [page](https://mckent05.github.io/Portfolio_Project/mobile.html)
+> To view this project live, please visit this [page](https://mckent05.github.io/Awesome_books/books.html)
 
 # Getting Started
```

Comment:
-[**_OPTIONAL_**] A non-professionally written `Getting Started` section warrants an invalidation of your Code Review Request. This getting started section is not descriptive enough for a normal user to get started with your project. This requirement is not really optional, work on it before submitting the next milestone of this project, please.

 Suggestion
- Tell the user how to clone your project in very simple terms
- After cloning, how do we navigate to the project's directory
- Lastly, how should the user run the project

Other issues to address:
- **_`About project`_**: This section is missing. It should detail what this project(app) is all about.
- **_`Contributions`_**: Tell us how we can contribute to this repo, what we have currently is not descriptive enough
- **_`Acknowledgement`_**: This section is missing
- Lastly, there is only one `Author` listed but there are two of you in this project, this also looks unprofessional, please work on it.

I have highlighted these so that we don't get issues with upcoming reviews, please work on the highlighted issues.